src/SAML2/XML/ds/KeyInfo.php

```diff
@@ -113,7 +86,7 @@ public function getInfo(): array
      * @param array $info
      * @return void
      */
-    public function setInfo(array $info): void
+    private function setInfo(array $info): void
     {
         $this->info = $info;
```

Comment:
We should probably have a check here to make sure the values in the array are only of the allowed classes (`\SAML2\XML\Chunk`, `\SAML2\XML\ds\KeyName` `\SAML2\XML\ds\X509Data`).

Reply:
Agreed